code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -478,6 +513,22 @@
 	return FALSE
 
 /mob/living/silicon/robot/bullet_act(var/obj/item/projectile/Proj)
+	if(HasTrait(CYBORG_TRAIT_DEFLECTIVE_BALLISTIC_ARMOR) && istype(Proj, /obj/item/projectile/bullet))
+		var/chance = 90
```

Comment:
Quite high.

Reply:
its admin only anyway , meant to be abused.

Reply:
Next PR adds it in a module for 7 plastic.